{
  try {
    _persistence.countByRootMessageId(RandomTestUtil.nextLong());
    _persistence.countByRootMessageId(0L);
  }
 catch (  Exception e) {
    Assert.fail(e.getMessage());
  }
}
